card/mod/pointer/set/abstract/01_pointer/edit.rb

```diff
@@ -89,7 +89,7 @@ def add_item_button
 
   view :multiselect do |_args|
     select_tag(
-      "pointer_multiselect",
+      "pointer_multiselect-#{unique_id}",
```

Comment:
I think this is breaking cukes.  Fwiw, those are awful cuke steps.  If this prompts us to make better high level steps, it will be a double gain!